static/lang/re-en.json

```diff
@@ -1734,6 +1734,12 @@
                 "Vampire": "Enable abilities that require a vampire target",
                 "Undead": "Enable abilities that require an undead target"
             },
+            "TrinityGeode": {
+                "Label": "Trinity Geode",
+                "Prompt": "Select cantrip or non-cantrip spell.",
+                "Cantrip": "Cantrip Spell",
+                "Non-Cantrip": "Non-Cantrip Spell"
```

Comment:
These 3 seem general enough that we probably want to just put them in a more generic place. I kind of want to do a large cleanup and organization of re-en.json to make it easier to find things and keep it from bloating further, that means generic text should maybe be in a different area. Haven't talked about it with the others yet though.

Reply:
Done.